### PR TITLE
Update clarity design config

### PR DESCRIPTION
**Pull request by Mavericker-1996**
# Pull request motivation(s)

update my  docsearch config

### What is the current behaviour?

*If the current behaviour is a bug, please provide all the steps to reproduce and screenshots with context.*

Our website's nb_hits was extremely low... which could only search few pages...

### What is the expected behaviour?

Increasing our website's nb_hits and return more search results

##### NB: Do you want to request a **feature** or report a **bug**?

**feature**

##### NB2: Any other feedback / questions ?

Thx :)

### CODE REVIEW

**Comment by s-pace** [comment]
Please @Mavericker-1996 [comply with the template we have set up](https://github.com/algolia/docsearch-configs/blob/master/.github/PULL_REQUEST_TEMPLATE). It will help us to better assist you and understand what you are trying to achieve. Thanks!

File: configs/alibaba-inc_clarity_design.json [hunk 1]
```diff
@@ -53,12 +57,12 @@
     "text": ".markdown-body p, .markdown-body li"
   },
   "js_render": true,
-  "js_wait": 1,
```
Comment (s-pace) [inline configs/alibaba-inc_clarity_design.json]: This value is needed given that your website takes time to render. Once you your website doesn't need client side rendering, you can remove this attribute. [See reference](https://community.algolia.com/docsearch/config-file.html#js_wait-optional)

Comment (Mavericker-1996) [reply configs/alibaba-inc_clarity_design.json]: Added :)

**Comment by Mavericker-1996** [comment]
> Please @Mavericker-1996 [comply with the template we have set up](https://github.com/algolia/docsearch-configs/blob/master/.github/PULL_REQUEST_TEMPLATE). It will help us to better assist you and understand what you are trying to achieve. Thanks!



> Please @Mavericker-1996 [comply with the template we have set up](https://github.com/algolia/docsearch-configs/blob/master/.github/PULL_REQUEST_TEMPLATE). It will help us to better assist you and understand what you are trying to achieve. Thanks!

Done~

**Comment by s-pace** [comment]
> Our website's nb_hits was extremely low... which could only search few pages...

`nb_hits` is only an output of the crawl. The issue here is that your website requires a heavy client side rendering which is not well handled by our crawler. Please make sure your website only rely on server side rendering [as per our recommendation.](https://community.algolia.com/docsearch/config-file.html#javascript-rendering)

**Comment by Mavericker-1996** [comment]
> > Our website's nb_hits was extremely low... which could only search few pages...
> 
> `nb_hits` is only an output of the crawl. The issue here is that your website requires a heavy client side rendering which is not well handled by our crawler. Please make sure your website only rely on server side rendering [as per our recommendation.](https://community.algolia.com/docsearch/config-file.html#javascript-rendering)

Actually we did SSR in our websites...

**Comment by s-pace** [comment]
Great news, I will update your config so that we do crawl your website without browser emulation.

**Comment by Mavericker-1996** [comment]
> Great news, I will update your config so that we do crawl your website without browser emulation.

Appreciated !

**Comment by s-pace** [comment]
@Mavericker-1996 crawl completed. You can try it out

**Comment by Mavericker-1996** [comment]
> @Mavericker-1996 crawl completed. You can try it out

It did work out ! Thank you very much !!!

**Comment by Mavericker-1996** [comment]
> @Mavericker-1996 crawl completed. You can try it out


It seems strange that all search results return with a "#container" anchor value, which leads our jumped url to the link with  "#container", It there a problem with our configuration ?

**Comment by s-pace** [comment]
@Mavericker-1996 please [implement this recommendation ](https://community.algolia.com/docsearch/tips.html#add-anchors-to-headings)

**Comment by Mavericker-1996** [comment]
> @Mavericker-1996 crawl completed. You can try it out

It failed to search again...
Is it because you deleted the two js_ fields?

**Comment by s-pace** [comment]
Just a pending crawl job with an outdated version of the config. Please give it a new try now, it will be fixed and will remains as it.

**Pull request merged**